applicationconfig/client/src/main/java/com/azure/applicationconfig/ConfigurationAsyncClient.java

```diff
@@ -37,41 +25,31 @@
  *     .build();
  * </pre>
  *
- * <p>View {@link ConfigurationAsyncClientBuilder this} for additional ways to construct the client.</p>
+ * <p>View {@link ConfigurationClientBuilder this} for additional ways to construct the client.</p>
  *
- * @see ConfigurationAsyncClientBuilder
+ * @see ConfigurationClientBuilder
  * @see ConfigurationClientCredentials
  */
-public final class ConfigurationAsyncClient extends ServiceClient {
-    private final ServiceLogger logger = new ServiceLogger(ConfigurationAsyncClient.class);
-
-    private static final String ETAG_ANY = "*";
-    private static final String RANGE_QUERY = "items=%s";
-
-    private final String serviceEndpoint;
-    private final ConfigurationService service;
+public final class ConfigurationAsyncClient {
+    private final ConfigurationRawClient client;
```

Comment:
Let's sync up on naming - not convinced 'Raw' is right, but let's see what else pops up.